tests/callbacks/test_stochastic_weight_avg.py

```diff
@@ -62,8 +69,23 @@ def configure_optimizers(self):
             },
         }
 
+    def training_epoch_end(self, _):
+        if not self.crash_after_epoch:
+            return
+        self._epoch_count += 1
+        if self._epoch_count >= self.crash_after_epoch:
+            raise RuntimeError("Crash test")
+
 
 class SwaTestCallback(StochasticWeightAveraging):
+    def __init__(self, *args, **kwargs):
+        if "resuming_from_epoch" in kwargs:
```

Comment:
Do we need `resuming_from_epoch `. Can't we use trainer.current_epoch ?

Reply:
Yes you're right, I've simplified this to use `trainer.current_epoch`